regulation-worker/internal/delete/api/api.go

```diff
@@ -106,22 +105,29 @@ func (api *APIManager) Delete(ctx context.Context, job model.Job, destination mo
 		return model.JobStatusFailed
 	}
 	jobStatus := getJobStatus(resp.StatusCode, jobResp)
-	pkgLogger.Debugf("[%v] JobStatus for %v: %v", destination.Name, destination.DestinationID, jobStatus)
+	pkgLogger.Debugf("[%v] JobStatus for %v: %v", destination.Name, job.DestinationID, jobStatus)
 
-	if isOAuthEnabled && isTokenExpired(jobResp) {
+	if isOAuthEnabled && isTokenExpired(jobResp) && attempts < 1 {
```

Comment:
I'd make the max retry attempts configurable.

Reply:
Made it configurable. Can you please verify ?